src/tribol/physics/CommonPlane.cpp

```diff
@@ -157,302 +156,301 @@ real ComputeGapRatePressure( ContactPlaneManager& cpMgr,
+              }
+
+              // compute each element spring stiffness. Pre-multiply the material modulus 
+              // (i.e. material stiffness) by each mesh's penalty scale
+              auto stiffness1 = pen_scale1 * mesh1.getElementData().m_mat_mod[ index1 ] / t1;
+              auto stiffness2 = pen_scale2 * mesh2.getElementData().m_mat_mod[ index2 ] / t2;
+              // compute the equivalent contact penalty spring stiffness per area
+              penalty_stiff_per_area = ComputePenaltyStiffnessPerArea( stiffness1, stiffness2 );
+              break;
+          }
+          default:
+              // no-op, quiet compiler
+              break;
+        } // end switch on kinematic penalty calculation option
+
+        ////////////////////////////////////////////////////
+        // Compute contact pressure(s) on current overlap // 
+        ////////////////////////////////////////////////////
+
+        // compute total pressure based on constraint type
+        RealT totalPressure = 0.;
+        plane.m_pressure = gap * penalty_stiff_per_area; // kinematic contribution
+        switch(pen_enfrc_options.constraint_type)
+        {
+          case KINEMATIC_AND_RATE:
+          {
+              // kinematic contribution
+              totalPressure += plane.m_pressure;
+              // add gap-rate contribution
+              totalPressure += 
+                ComputeGapRatePressure( plane, mesh1, mesh2, penalty_stiff_per_area,
+                                        pen_enfrc_options.rate_calculation );
+              break;
+          }
+          case KINEMATIC:
+              // kinematic gap pressure contribution  only
+              totalPressure += plane.m_pressure;
+              break;
+          default:
+              // no-op
+              break;
+        } // end switch on registered penalty enforcement option
+
+        // debug prints. Comment out for now, but keep for future common plane 
+        // debugging
+  //         SLIC_DEBUG("gap: " << gap);
+  //         SLIC_DEBUG("area: " << A);
+  //         SLIC_DEBUG("penalty stiffness: " << penalty_stiff_per_area);
+  //         SLIC_DEBUG("pressure: " << cpManager.m_pressure[ cpID ]);
+
+        ///////////////////////////////////////////
+        // create surface contact element struct //
+        ///////////////////////////////////////////
+
+        // construct array of nodal coordinates
+        constexpr int max_dim = 3;
+        constexpr int max_nodes_per_face = 4;
+        constexpr int max_nodes_per_overlap = 8;
+        RealT xf1[ max_dim * max_nodes_per_face ];
+        RealT xf2[ max_dim * max_nodes_per_face ];
+        RealT xVert[ max_dim * max_nodes_per_overlap ];  
+        int dim = cs_view.spatialDimension();
+        int num_nodes_per_face = mesh1.numberOfNodesPerElement();
+        initRealArray( xf1, dim * num_nodes_per_face, 0. );
+        initRealArray( xf2, dim * num_nodes_per_face, 0. );
+        auto xVert_size = 4;
```

Comment:
Perhaps comment that xVert_size = 4 and numPolyVert = 2 are for 2D and if 3D, they are modified just below here.